src/app/pokedex/pokedex-tile.tsx

```diff
@@ -0,0 +1,19 @@
+import Image from "next/image";
+import { Pokemon } from "@/models/pokemon";
+import "./pokedex-tile.css";
+
+export default function PokedexTile({ pokemon }: { pokemon: Pokemon }) {
+    const iconUrl = `https://projectpokemon.org/images/sprites-models/sv-sprites-home/${pokemon.national_pokedex_number.toString().padStart(4, "0")}.png`;
+    const type1CSS = `var(--type_${pokemon.type[0]})`;
```

Comment:
better name than type 1 possible?

Reply:
Yeah fair.  I'll swap to firstTypeCss and secondTypeCss